frontend/src/routes/Infrastructure/Automations/AnsibleAutomationsForm.tsx

```diff
@@ -223,6 +244,7 @@ export function AnsibleAutomationsForm(props: {
           prehook: upgradePreJobs,
           posthook: upgradePostJobs,
         },
+        inventory: ansibleInventory,
```

Comment:
```suggestion
        ...(ansibleInventory ? { inventory: ansibleInventory } : {}),
```
This will prevent the key from showing in the YAML when no inventory is selected.